### PR TITLE
Create a new section in the User Manual for Taipy Studio

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #317.

### CODE REVIEW

**Review by FredLL-Avaiga — APPROVED** [review]

File: docs/manuals/studio/index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Taipy Studio
+
+Taipy Studio is an extension to [Visual Studio Code](https://code.visualstudio.com/)
+that significantly accelerates the development of Taipy applications.
+
+This tool addresses two major areas:
+
+- Building configurations.<br/>
+    An application that relies on [Taipy Core](../core/) needs to build a Taipy
```
Comment (jrobinAV) [inline docs/manuals/studio/index.md]:
```suggestion
    An application that relies on [Taipy Core](../core/) functionalities needs to build a Taipy
```

File: docs/manuals/studio/index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    [configuration](../core/config/) where DataNodes, Tasks, etc. are declared.
```
Comment (jrobinAV) [inline docs/manuals/studio/index.md]:
```suggestion
    [configuration](../core/config/) where DataNode configs, Task configs, etc. are declared.
```

File: docs/manuals/studio/index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    Running the application usually starts by creating [entities](../core/entities/),
+    which depend on configuration elements.
+
+    Taipy Studio provides a graphical editor where you can create and configure
+    the elements of your configurations stored in toml files that the
+    application can use at run-time. The editor can represent pipelines and
+    scenarios as graphs, where nodes are DataNodes or Tasks.
```
Comment (jrobinAV) [inline docs/manuals/studio/index.md]:
```suggestion
    Taipy Studio provides a graphical editor where you can create and configure
    the elements of your configurations stored in toml files that your Taipy
    application can load at run-time. The editor can represent scenarios as 
    graphs, where nodes are DataNodes or Tasks.
```

I would remove the pipeline term here 'cause we want to put more light on Scenario.
In particular, we want to progressively hide pipelineConfigs. We won't remove it 
completely now, it must be progressive, but at least the documentation can drive this.

File: docs/manuals/studio/index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+- the current project contains any file with a `.py` extension (then
+  these Python source files may contain strings that define Taipy GUI
+  pages);
+- you explicitly open the *Taipy Configs* view (using the "View > Open View..."
+  menu option).
+
+Depending on your task at hand, you may want to read more on
+[building Taipy configurations](config.md) or on [Taipy GUI Markdown support](gui.md).
+
+## [:material-arrow-right: Building Taipy configurations](config.md)
```
Comment (jrobinAV) [inline docs/manuals/studio/index.md]:
```suggestion
## [:material-arrow-right: Building configurations](config.md)
```

File: docs/manuals/studio/index.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+## [:material-arrow-right: Taipy GUI Markdown support](gui.md)
```
Comment (jrobinAV) [inline docs/manuals/studio/index.md]:
```suggestion
## [:material-arrow-right: GUI Markdown support](gui.md)
```

**Review by jrobinAV — APPROVED** [review]
A few remarks. None is really blocking for me.

File: docs/manuals/studio/index.md (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/studio/index.md]: Ok.

File: docs/manuals/studio/index.md (hunk 3 above)
Comment (FabienLelaquais) [reply docs/manuals/studio/index.md]: Kind of done.

File: docs/manuals/studio/index.md (hunk 4 above)
Comment (FabienLelaquais) [reply docs/manuals/studio/index.md]: 👍

File: docs/manuals/studio/index.md (hunk 5 above)
Comment (FabienLelaquais) [reply docs/manuals/studio/index.md]: 👍

**Pull request merged**